geo-test-fixtures/Cargo.toml

```diff
@@ -5,5 +5,5 @@ edition = "2021"
 publish = false
 
 [dependencies]
-wkt = { version = "0.10.0", default-features = false }
+wkt = { version = "0.10.0", features = ["geo-types"], default-features = false }
```

Comment:
This is a bit unfortunate, since `wkt` being used here without `geo-types` was one of the main arguments against moving `wkt` into the `geo` repository. That's why `line_string` below is used instead of the `wkt` feature.

Reply:
I don't think I ever made that argument, but this does introduce a circular dependency, which I think is the concern, right? 

There's been a lot of ink spilled about circular dev-dependencies in georust. I get why it's not conceptually ideal, but in practice I've found it pretty easy to add a temporary entry to `[patch.crates-io]` while I have a change in flight.

@lnicola  - can you expound on what you'd like to see happen here? Is it your preference that I copy and paste the conversion of WKT into a geo-type::Polygon into geo-test-fixtures? 

A bigger change could be to move the integrations into the geo-types crate (behind a `wkt` feature) like we did with `rstar`. What would you think of that?

Reply:
> A bigger change could be to move the integrations into the geo-types crate (behind a wkt feature) like we did with rstar. What would you think of that?

To add to this, IIRC we talked about doing something like this in the past, but I was concerned it would mean "breaking" geo-types semver whenever we require a newer breaking version of `wkt`. But we've since learned a nicer way to handle this - as exemplified with our simultaneous rstar-0.8 and rstar-0.9 integrations, so I have less reservations about it than I might have a year ago.

Reply:
> I don't think I ever made that argument, but this does introduce a circular dependency, which I think is the concern, right?

No, I did :sweat_smile:. And yes, it adds an awkward (though not circular) dependency: it pulls in another version of `geo-types`, so the types returned by `wkt` will not always be compatible with those in the current repo.

> Is it your preference that I copy and paste the conversion of WKT into a geo-type::Polygon into geo-test-fixtures?

That's what I did for `line_string`. I think it's not too bad.

> A bigger change could be to move the integrations into the geo-types

That would work. I don't have any strong opinion about which direction it's better to do this in.

Reply:
>> Is it your preference that I copy and paste the conversion of WKT into a geo-type::Polygon into geo-test-fixtures?
>
> That's what I did for line_string. I think it's not too bad.

Alright, I've done that for now since it seems easy and localized. I'm not overly worried about hacks in our fixture loading code.

> A bigger change could be to move the integrations into the geo-types

I might follow up one day with this...

Reply:
Thanks, looks good, but please squash at the end to keep only one copy of the WKTs.

Reply:
Done - I squashed the fixture changes into the first commit.